api/tools/src/main/java/org/pmiops/workbench/tools/elastic/ElasticSearchIndexer.java

```diff
@@ -306,9 +306,11 @@ private void createIndex(CommandLine opts) throws IOException, InterruptedExcept
   }
 
   private String getPersonBigQuerySQL(String bqDataset, int inverseProb) throws IOException {
+    // Synth dataset doesn't contain sex_at_birth, so we use gender as a fix
```

Comment:
Owch. Will check for an update on synthetic data this week. We need to be able to rerun the latest curation pipeline at a bare minimum even if we can't easily make data improvements yet.

Reply:
Agreed.